force_wfmanager/left_side_pane/side_pane.py

```diff
@@ -43,13 +43,15 @@ class SidePane(TraitsDockPane):
     #: Used when the computation is running
     enabled = Bool(True)
```

Comment:
We will have to come back and fix this to "ui_enabled" to clarify. Not now, maybe open an issue for it.